base/process.jl

```diff
@@ -2,22 +2,76 @@
 
 abstract AbstractCmd
 
+# libuv process option flags
+const UV_PROCESS_WINDOWS_VERBATIM_ARGUMENTS = UInt8(1 << 2)
+const UV_PROCESS_DETACHED = UInt8(1 << 3)
+const UV_PROCESS_WINDOWS_HIDE = UInt8(1 << 4)
+
 immutable Cmd <: AbstractCmd
     exec::Vector{ByteString}
     ignorestatus::Bool
-    detach::Bool
+    flags::UInt8 # libuv process flags
```

Comment:
If we change it to support additional flags, and at some distant future time libuv uses more than 8, we can always change the `flags` variable.

Unless there is some reason that we don't want `sizeof(Cmd)` to ever change?